query/src/configs/config_storage.rs

```diff
@@ -161,6 +161,19 @@ impl fmt::Debug for AzblobStorageConfig {
     }
 }
 
+#[derive(Clone, PartialEq, Serialize, Deserialize, Default, Args, Debug)]
+#[serde(default)]
+pub struct HdfsConfig {
+    #[clap(long = "storage-hdfs-name-node", default_value_t)]
+    pub name_node: String,
```

Comment:
What's the name_node URI look like? Is there any spec for it in OpenDAL.

Reply:
> What's the name_node URI look like? Is there any spec for it in OpenDAL.

Docs: https://docs.rs/opendal/0.6.1/opendal/services/hdfs/struct.Builder.html

They are the same as how hdfs java clients use.